user-guides/data-farming/how-to-estimate-apy.md

```diff
@@ -0,0 +1,41 @@
+---
+description: >-
+  Learn how to use basic math and a simple spreadsheet to estimate your
+  Passive Rewards APY and get deeper into Active Rewards APY.
+---
+
+<figure><img src="../../.gitbook/assets/gif/mafs.gif" alt=""><figcaption><p>K.I.S.S.</p></figcaption></figure>
+
+# veOCEAN and your APY
+
+Rewards are earned by users that hold and use their veOCEAN to help the protocol improve and grow.  
+
+To receive competitve short-term APYs, veOCEAN requires users to make long-term commitments.
+
+First, make sure that you [understand the basics of APY](../../rewards/df-veocean.md#veocean-and-your-apy) and how veOCEAN works.
```

Comment:
the link is not working

Reply:
this is working locally so not sure what's the issue